examples/problems/800_score_genes_for_marginals.ipynb

```diff
@@ -207,7 +207,13 @@
    "source": [
```

Comment:
<p>add that delta t is automatically inferred from <a href="https://moscot.readthedocs.io/en/latest/genapi/moscot.problems.time.TemporalProblem.temporal_key.html#moscot.problems.time.TemporalProblem.temporal_key" target="_blank">https://moscot.readthedocs.io/en/latest/genapi/moscot.problems.time.TemporalProblem.temporal_key.html#moscot.problems.time.TemporalProblem.temporal_key</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>